src/data/embedding.py

```
@@ -1 +1,117 @@
+                        for cell in row:
+                            if cell:
+                                texto += str(cell) + " "
+                        texto += "\n"
+            except ImportError:
+                texto = f"Archivo XLSX: {blob.name} (necesita openpyxl)"
+        else:
+            # Para otros archivos, intentar leer como texto
+            try:
+                with open(local_path, 'r', encoding='utf-8') as file:
+                    texto = file.read()
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)